test/PangenomeFileUtil_server_test.py

```diff
@@ -51,73 +87,57 @@ def tearDownClass(cls):
-        ws_name =  'msneddon:1450308782878' #self.getWsName()
-        #with open(os.path.join('data','dummy_pangenome.json')) as data_file:    
-        #    pgData = json.load(data_file)
-        #ws = self.getWsClient();
-        #info = ws.save_objects({'workspace':self.getWsName(), 'objects':[
-        #            {
-        #                'type':'KBaseGenomes.Pangenome',
-        #                'data':pgData,
-        #                'name':obj_name
-        #            }]})
-        #print('saved dummy pangenome')
-        #pprint(info)
-
-        # attempt to download as tsv file
-        res = pfut.pangenome_to_tsv_file(self.getContext(),
-                                        {
-                                            'pangenome_name':obj_name,
-                                            'workspace_name':ws_name,
-                                        })
+    def test_to_tsv(self):
+        res = self.getImpl().pangenome_to_tsv_file(self.getContext(), {
+                'pangenome_name': self.obj_name,
+                'workspace_name': self.wsName,
+            })
         pprint(res)
-
-        # attempt to create the download package
-        res = pfut.export_pangenome_as_tsv_file(self.getContext(),
-                                        {
-                                            'input_ref':ws_name + '/' + obj_name
-                                        })
+        # test bad input
+        with self.assertRaises(ValueError):
+            self.getImpl().pangenome_to_tsv_file(self.getContext(), {
+                'pangenome_ref': self.wsName + '/' + self.obj_name
+            })
+
+    def test_to_excel(self):
+        res = self.getImpl().pangenome_to_excel_file(self.getContext(), {
+                'pangenome_name': self.obj_name,
+                'workspace_name': self.wsName,
+            })
         pprint(res)
-        # need a little cleanup because we try to run on this object twice- normally
-        # the sdk will give you a fresh directory so this is not a problem.
-        shutil.rmtree(os.path.join(self.__class__.cfg['scratch'],obj_name))
-
-
-
-        res = pfut.pangenome_to_excel_file(self.getContext(),
-                                        {
-                                            'pangenome_name':obj_name,
-                                            'workspace_name':ws_name,
-                                        })
-        # attempt to create the download package
-        res = pfut.export_pangenome_as_excel_file(self.getContext(),
-                                        {
-                                            'input_ref':ws_name + '/' + obj_name
-                                        })
+        # test bad input
+        with self.assertRaises(ValueError):
+            self.getImpl().pangenome_to_excel_file(self.getContext(), {
+                'pangenome_ref': self.wsName + '/' + self.obj_name
+            })
+
+    def test_export_tsv(self):
```

Comment:
same here. It checked something has saved to shock. But ignore what content has been saved.